jetson/percep_obs_detect/.gitignore

```diff
@@ -0,0 +1,10 @@
+build/
```

Comment:
Move these to the main .gitignore at the very front of the workspace. You can make a section for perception